glean-core/src/internal_metrics.rs

```diff
@@ -112,6 +116,23 @@ impl AdditionalMetrics {
                 },
                 TimeUnit::Millisecond,
             ),
+
+            // This uses a `send_in_pings` that contains "all-ping".
+            // This works because all of our other current "all-pings" metrics
+            // have special handling internally and are not actually processed
+            // into a store quite like this identifier is.
+            //
+            // This could become an issue if we ever decide to start generating
+            // code from the internal Glean metrics.yaml (there aren't currently
+            // any plans for this).
```

Comment:
I heard those words before 🤔

Reply:
Just wanted to leave them somewhere I could easily find them if/when this breaks here :)